.github/workflows/osx.yml

```diff
@@ -29,7 +29,7 @@ on:
         default: false
 
 jobs:
-  build:
+  build_osx_tests:
```

Comment:
thanks for this but also why n ot match what you did below and just do `build_osx`